### ISSUE TITLE
`leanproject check` fails with error

### ISSUE THREAD

**Issue by gebner** [leanprover-community/mathlib-tools#27]
```
$ elan default 3.4.2
$ leanproject get mathlib
$ cd mathlib
$ lean --version
Lean (version 3.6.1, commit 97f11c47b944, Release)
$ lean -p
{
  "is_user_leanpkg_path": false,
  "leanpkg_path_file": "/home/gebner/tmp/mathlib/leanpkg.path",
  "path": [
    "/home/gebner/.elan/toolchains/leanprover-community-lean-3.6.1/bin/../library",
    "/home/gebner/.elan/toolchains/leanprover-community-lean-3.6.1/bin/../lib/lean/library",
    "/home/gebner/tmp/mathlib/./src"
  ]
}
$ leanproject check
[Errno 2] No such file or directory: '/home/gebner/.elan/toolchains/3.5c/tmp/bench30.olean'
```

I'm not sure why it searches in `/home/gebner/.elan/toolchains/3.5c` at all.

**Comment by PatrickMassot** [comment]
This was a bit lazy from me. `leanproject check` checks everything in `$HOME/.elan/toolchains`. I wasn't expecting people to store junk there (because I'm not playing with Lean itself).

**Comment by PatrickMassot** [comment]
Fixed in a856735
